discussion/validators.py

```diff
@@ -0,0 +1,19 @@
+import os
+from django.core.exceptions import ValidationError
+def validate_file_extension(file):
+    """
+    Validates the file extension and type
+    """
+
+    valid_extensions = ['.pdf', '.png', '.jpg']
+    valid_content_types = ['application/pdf', 'image/png', 'image/jpg']
+
+    ext = os.path.splitext(file.name)[1]  # [0] returns path+filename
```

Comment:
The following may be be more clear:
`_file_path, ext = os.path.splittext(file.name)`

Reply:
done!